mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -1051,7 +1080,7 @@ public void searchWithTextShouldReturnMailsWhenCcMatches() throws Exception {
     @Test
     public void searchWithTextShouldReturnMailsWhenBccMatches() throws Exception {
         Assume.assumeTrue(storeMailboxManager.getSupportedSearchCapabilities().contains(MailboxManager.SearchCapabilities.Text));
-        SearchQuery searchQuery = new SearchQuery(SearchQuery.textContains("any.com"));
+        SearchQuery searchQuery = new SearchQuery(SearchQuery.textContains("any@any.com"));
```

Comment:
so it's no more possible to search for any to find any@example.com? And so search for a domain to have all emails related to a customer?

Reply:
We may achieve this with search analyzer and index analyzer. 

If we index using standard analyzer on a first field and email analyzer on another one, then execute or search query on both with only email analyzer, I guess it works.

Reply:
DONE